proposals/left-right-join-in-query-expressions.md

```diff
@@ -0,0 +1,235 @@
+# `left join` and `right join` query expression clauses
+
+Champion issue: https://github.com/dotnet/csharplang/issues/8947
+
+## Summary
+[summary]: #summary
+
+Introduce `left join` and `right join` clauses into LINQ query expression syntax, e.g.:
```

Comment:
*perhaps* tweak the wording to indicate that this is more about updating the existing 'join' clause to support an optional `left/right` modifier to tweak its behavior.  Conveys that this is more minor than adding full new clauses, and also conveys that the joins are otherwise the same with what we've already shpped.